orchestration/templates/process-table.yaml

```diff
@@ -98,3 +98,102 @@ spec:
                   value: '{{ "{{inputs.parameters.gcs-prefix}}/old-ids/{{inputs.parameters.table-name}}" }}'
                 - name: table-name
                   value: '{{ "{{inputs.parameters.table-name}}" }}'
+          # Generate BQ tables for ClinGen
+          # new rows
+          - name: generate-new-rows-table
+            dependencies: [diff-table]
+            template: query-full-rows
+            arguments:
+              parameters:
+                - name: bq-project
+                  value: '{{ "{{inputs.parameters.staging-bq-project}}" }}'
+                - name: bq-dataset
+                  value: '{{ "{{inputs.parameters.staging-bq-dataset}}" }}'
+                - name: input-table
+                  value: '{{ "{{tasks.diff-table.outputs.parameters.join-table-name}}" }}'
+                - name: table-name
+                  value: '{{ "{{inputs.parameters.table-name}}" }}'
+                - name: query-type
+                  value: new_rows
+          # updated rows
+          #- name: generate-updated-rows-table
+          #  dependencies: [diff-table]
+          #  template: query-full-rows
+          # arguments:
+          #   parameters:
+          #     - name: bq-project
+          #       value: '{{ "{{inputs.parameters.staging-bq-project}}" }}'
+          #     - name: bq-dataset
+          #       value: '{{ "{{inputs.parameters.staging-bq-dataset}}" }}'
+          #     - name: input-table
+          #       value: '{{ "{{tasks.diff-table.outputs.parameters.join-table-name}}" }}'
+          #     - name: table-name
+          #       value: '{{ "{{inputs.parameters.table-name}}" }}'
+          #     - name: query-type
+          #       value: update
+          # deleted rows - TODO
+          #- name: generate-deleted-rows-table
+          #  dependencies: [diff-table]
+          #  template: query-row-ids
+          # arguments:
+          #   parameters:
+          #     - name: bq-project
+          #       value: '{{ "{{inputs.parameters.staging-bq-project}}" }}'
+          #     - name: bq-dataset
+          #       value: '{{ "{{inputs.parameters.staging-bq-dataset}}" }}'
+          #     - name: input-table
+          #       value: '{{ "{{tasks.diff-table.outputs.parameters.join-table-name}}" }}'
+          #     - name: table-name
+          #       value: '{{ "{{inputs.parameters.table-name}}" }}'
+          #     - name: query-type
+          #       value: update
+    # Subtemplates
+    # query-full-rows will be generic for both the new rows + updates
+    - name: query-full-rows
+      inputs:
+        parameters:
+          - name: table-name
+          - name: join-table-name
+          - name: bq-project
+          - name: bq-dataset
+          - name: query-type
+      {{- include "argo.retry" . | indent 6 }}
+      script:
+        # setting env variables for the bash script
```

Comment:
Nitpick: This comment is redundant with the syntax of Argo itself. A better comment would explain _why_ we inject the variables via the environment.